compiler/AST/expr.cpp

```diff
@@ -3722,9 +3722,67 @@ CallExpr::insertAtTail(BaseAST* ast) {
 }
 
 
-FnSymbol* CallExpr::isResolved(void) {
-  SymExpr* base = toSymExpr(baseExpr);
-  return base ? toFnSymbol(base->var) : NULL;
+bool CallExpr::isEmpty() const {
+  return primitive == NULL && baseExpr == NULL;
+}
+
+
+// MDN 2016/01/29: This will become a predicate
+FnSymbol* CallExpr::isResolved() const {
```

Comment:
Right; that's the next series of steps.

1) Go through the code base converting uses of isResolved() to resolvedFunction() where appropriate and in
manageable pieces

2) Convert isResolved() to return a bool